qiskit/opflow/primitive_ops/tapered_pauli_sum_op.py

```diff
@@ -356,32 +356,41 @@ def find_Z2_symmetries(cls, operator: PauliSumOp) -> "Z2Symmetries":
 
         return cls(pauli_symmetries, sq_paulis, sq_list, None)
 
-    def taper(self, operator: PauliSumOp) -> OperatorBase:
+    def convert_clifford(self, operator: PauliSumOp) -> OperatorBase:
```

Comment:
Was there a reason to drop the checks/error raising in each of these public methods. Maybe it was a concern that it would be done twice when calling these back to back in taper (which would not be an issue really aside from a little redundancy). But here either of these methods can be called now and they will proceed with no checks.

Reply:
Yes it was the redundancy that bothered me. I'll just double the check then, thanks